students/hirot_directory/fib_hiroTakechi.py

```diff
@@ -21,14 +22,17 @@ def fib(n):
 	assert fib(3) == 2
 	print(fib(10))
 
+'''
+
 # the codes that I worked on after class: 
 
+'''
 ## code 1:
 def fibonacci(n, a=1, b=0):
     return b if n < 1 else fibonacci(n - 1, a + b, a)
 
 print(fibonacci(7))
-
+'''
 
 ## code 2:
 class Program():
```

Comment:
So this works, but is rally not how you'd want to do it.

In general, we don't put functions in classes by themselves at all in python.

We'll get to classes in a couple weeks.

Reply:
Hi Chris,

OK, then I will wait to learn about the class for a few weeks. About PR, I didn't see the same response from Github when I committed the revision so I wasn't sure of whether it was working but it did work. That's good news.

Thanks!
Hiro